.github/workflows/coverage-check.yml

```diff
@@ -59,28 +59,25 @@ jobs:
                   "src/dollar/utils/*" \
                   "test/*" \
 
-
           # Generate summary
           COVERAGE_DEVELOPMENT_OUTPUT=$(lcov \
               --rc lcov_branch_coverage=1 \
               --list filtered-lcov.info)
 
           echo COVERAGE=$(echo "${COVERAGE_DEVELOPMENT_OUTPUT}" | tail -n 1 | cut -d % -f 1 | cut -d \| -f 2) >> $GITHUB_OUTPUT
 
+      - name: Delete development branch folder
+        run: rm -rf development
+
       - name: Checkout code in PR branch
         uses: actions/checkout@v3
 
-      - name: Update Forge Dependencies
-        working-directory: packages/contracts
-        run: forge update lib/forge-std
-
       - name: Get PR branch coverage
         id: coverage-pr
+        working-directory: packages/contracts
         run: |
-          cd ./packages/contracts
-
           # generates lcov.info
-          forge coverage --report lcov
+          forge build && forge coverage --report lcov
 
           # Foundry uses relative paths but Hardhat uses absolute paths.
```

Comment:
@gitcoindev can you update this to have it good on this same PR? we do not use hardhat on this repo, appreciate it

Reply:
Done, see commit 5b754a4d03f65f88b8ffde8b95e978a8358ea381